documentation/modules/security/ref-formats-kafka-listener-certificates.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='ref-formats-kafka-listener-certificates-{context}']
+
+= SAN certificates for Kafka listeners
+
+If you provide your own xref:kafka-listener-certificates-{context}[Kafka listener certificates], you must use a Subject Alternative Name (SAN) certificate for each listener. The SAN certificate must specify hostnames for:
+
+* All of the Kafka brokers in your cluster
+
+* The Kafka cluster bootstrap service
+
+The Kafka broker hostnames can contain wildcard names.
+
+Using SAN certificates is required for TLS hostname verification.
```

Comment:
```suggestion
Using certificates with the coorrect SANs is required for TLS hostname verification.
```

Reply:
I'll remove this sentence because we now mention TLS hostname verification right at the start.